### PR TITLE
new icon: nodemon (original, plain, line)

### PR DESCRIPTION
**Pull request by PantheraRed**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

https://nodemon.io/
https://github.com/remy/nodemon/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -5216,6 +5216,28 @@
...
         "aliases": []
     },
+    {
+        "name": "nodemon",
+        "altnames": [
+            "nodemonjs"
+        ],
+        "tags": [
+            "nodejs"
+        ],
```
Comment (lunatic-fox) [inline devicon.json:5226]: Just a little suggestion.
```suggestion
        "tags": [
            "nodejs",
            "tool"
        ],
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Hello there!
Icons seem flawless. 😀

File: devicon.json (hunk 1 above)
Comment (PantheraRed) [reply devicon.json:5226]: Would like to hear @Snailedlt's opinion on it as well.

Comment (Snailedlt) [reply devicon.json:5226]: Agreed :)
But can we also add the Javascript tag? Nodemon is built with Javascript, and for Javascript
```suggestion
        "tags": [
            "nodejs",
            "tool",
            "javascript"
        ],
```

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for this contribution! 🚀

**Review by Snailedlt — APPROVED** [review]
Awesome! :heavy_check_mark: 
One of the fastest PR approvals every. Thank you for your contribution! :100:

**Pull request merged**